letta/server/rest_api/routers/v1/organizations.py

```diff
@@ -38,8 +38,7 @@ def create_org(
     """
     Create a new org in the database
     """
-
-    org = server.create_organization(request)
+    org = server.organization_manager.create_organization(request)
```

Comment:
You can ideally forgo the organization_manger completely and just import the organization directly either here or in server.py if you only want to centralize logic in server. Generally the pattern is to create one db_session object through the whole request (commonly done via [Depends](https://fastapi.tiangolo.com/tutorial/dependencies/), though you could create it in the server instance as part of `get_letta_server` and reference it via `self.db_session` in server.create_organization